src/components/hv-list/types.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) Garuda Labs, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+export type State = {
+  refreshing: boolean;
+};
+
+// https://reactnative.dev/docs/flatlist#scrolltoindex
+export type ScrollParams = {
+  animated?: boolean | null | undefined;
```

Comment:
I don't think we need both the question mark and the ' | undefined '

Reply:
the ? indicates optionality, which means the following is valid:
```js
{ index: 0 }
```

Reply:
I thought it was interchangeable with `undefined`. My mistake